vue.YAML-tmLanguage

```diff
@@ -174,7 +174,7 @@ patterns:
       '1': {name: punctuation.definition.tag.end.html}
     end: (?=</(?i:style))
     patterns:
-    - include: source.css.scss
+    - include: source.scss
```

Comment:
I believe it should be `source.scss` but can someone verify if it does not break on Sublime?